streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -403,4 +416,69 @@ public void shouldThrowExceptionWhenTopicNamesAreNull() {
         builder.stream(Arrays.asList(null, null));
         builder.build();
     }
+
+    @Test
+    public void shouldUseSpecifiedNameForStreamSourceProcessor() {
+        final String expected = "source-processor";
+        builder.stream(STREAM_TOPIC, Consumed.as(expected));
+        builder.stream(STREAM_TOPIC_TWO);
+        builder.build();
```

Comment:
Why do we need to call `build()` here? (similar below)

Reply:
It seems the build method  generates and adds the processors  into the InternalTopologyBuilder instance. If we don't invoke this method the TopologyProcessor is empty.